charts/gateway-helm/templates/leader-election-rbac.yaml

```diff
@@ -44,7 +45,8 @@ metadata:
   name: {{ include "eg.fullname" . }}-leader-election-rolebinding
   namespace: '{{ .Release.Namespace }}'
   labels:
-  {{- include "eg.labels" . | nindent 4 }}
+    {{- include "eg.labels" . | nindent 4 }}
```

Comment:
looks like the idententation is changing for all these files, was it broken earlier ?